<launch>
	<node pkg="i2c_imu" type="i2c_imu_node" name="i2c_imu" output="screen" >
		<rosparam file="$(find i2c_imu)/param/imu_9250.yaml"/>
	</node>        
</launch>
